### PR TITLE
perfect square

### PR DESCRIPTION
**Pull request by shellhub**
* add perfect square number

### CODE REVIEW

**Comment by shellhub** [comment]
@cclauss Please review!

File: maths/perfect_square.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import math
+
+
+def perfect_square(num):
```
Comment (cclauss) [inline maths/perfect_square.py]: https://docs.python.org/3/library/typing.html

Comment (shellhub) [reply maths/perfect_square.py]: Like below code?
``` python
def perfect_square(num: int) -> bool:
```

Comment (cclauss) [reply maths/perfect_square.py]: Perfect!

Comment (shellhub) [reply maths/perfect_square.py]: updated! Please review again!

**Pull request merged**